src/ga4gh/vrs/_internal/models.py

```diff
@@ -271,6 +278,12 @@ class LiteralSequenceExpression(_ValueObject):
     )
     sequence: SequenceString = Field(..., description='the literal sequence')
 
+    class ga4gh(_Ga4ghIdentifiableObject.ga4gh):
```

Comment:
Adding inheritance from `_Ga4ghIdentifiableObject.ga4gh` made `LiteralSequenceExpression.ga4gh.identifiable=True` instead of undefined. `LiteralSequenceExpression` should be stripped to these keys but not identifiable (meaning not digested and replaced with its digest during `ga4gh_serialize`).

Reply:
I read through the method and agree this will need refactoring, especially as we have adjusted our serialization strategy to support `ga4gh.keys` in both identifiable and non-identifiable objects.